test/integration/package.json

```diff
@@ -5,14 +5,13 @@
   "license": "BSD",
   "main": "index.js",
   "dependencies": {
-    "mapbox-gl-styles": "2.0.2",
     "colors": "^1.1.2",
     "d3-queue": "^3.0.3",
     "diff": "^3.0.0",
     "lodash": "^4.17.4",
+    "mapbox-gl-styles": "2.0.2",
     "pixelmatch": "^4.0.2",
     "pngjs": "^3.0.0",
-    "st": "^1.1.0",
-    "shuffle-array": "^1.0.1"
```

Comment:
The `shuffle-seed` dependency needs to be duplicated here, due to how we manage dependencies in native.